src/JuMP.jl

```diff
@@ -990,8 +990,8 @@ Can be unset using [`unset_time_limit_sec`](@ref) or with `limit` set to
 
 See also: [`unset_time_limit_sec`](@ref), [`time_limit_sec`](@ref).
 """
-function set_time_limit_sec(model::Model, limit)
-    return MOI.set(model, MOI.TimeLimitSec(), limit)
+function set_time_limit_sec(model::Model, limit::Real)
```

Comment:
`limit` can be `nothing` as well, see CI failure

Reply:
People should use `unset_time_limit_sec` in that case. But yeah, I'll change.